### DISCUSSION TITLE
Importing data from a csv

### DISCUSSION THREAD

**Discussion by BradleyTesting** [ocornut/imgui#6050] [New Users Build/Link/Run issues ONLY!]
Hey, I have been trying and failing to be able to import data from a csv, save it in the program for use in a regression model and then have the option to see the data in a pop up window. Does anyone know if there is a way to do this?

**Comment by PathogenDavid** [comment]
Processing CSV files is not something Dear ImGui will handle for you, you will need to do it yourself.

As far as displaying it goes, you'd probably want to use a table. There are examples in the demo.

Also the discussions category is not for general questions. Please see [the pinned topic](https://github.com/ocornut/imgui/discussions/4497).
